src/iris/api.py

```diff
@@ -3684,11 +3693,14 @@ def get_api(config):
     debug = False
     if config['server'].get('disable_auth'):
         debug = True
-    master_sender = config['sender'].get('master_sender', config['sender'])
-    master_sender_addr = (master_sender['host'], master_sender['port'])
+
+    default_master_sender = config['sender'].get('master_sender', config['sender'])
```

Comment:
Can we avoid .get() tests on the config by ensuring that it is always there when generating the config dictionary?